<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionDialog</class>
 <widget class="QDialog" name="SessionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Session</string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="sourceSection">
     <property name="title">
      <string>Source</string>
     </property>
     <layout class="QGridLayout" name="sourceSectionLayout" columnstretch="0,1">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="localSessionRadio">
        <property name="text">
         <string>Start &amp;Local Session</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="remoteSessionRadio">
        <property name="text">
         <string>Connect to &amp;Remote Session</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="hostLabel">
        <property name="text">
         <string>&amp;Host:</string>
        </property>
        <property name="buddy">
         <cstring>hostEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="hostEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>&amp;Port:</string>
        </property>
        <property name="buddy">
         <cstring>portSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="portSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="authCheck">
        <property name="text">
         <string>&amp;Authentication required</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>&amp;Username:</string>
        </property>
        <property name="buddy">
         <cstring>usernameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="usernameEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Pass&amp;word:</string>
        </property>
        <property name="buddy">
         <cstring>passwordEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="rpcLabel">
        <property name="text">
         <string>RPC URL pa&amp;th:</string>
        </property>
        <property name="buddy">
         <cstring>rpcEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="rpcEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtons</sender>
   <signal>accepted()</signal>
   <receiver>SessionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>SessionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
